pypolygames/utils/helpers.py

```diff
@@ -19,7 +19,7 @@ def generate_random_seeds(seed: int) -> Iterator[int]:
     # generate random seeds
     generator = random.Random(seed)
     while True:
-        yield generator.randint(0, 2 ** 31 - 1)
```

Comment:
I think this is the correct formatting, even for exponentiation

Reply:
We're just conforming to what `black==23.1.0` says. Any other formatting fails the CI. If you don't like it I think a previous `black` version had another format.